tests/e2e/specs/miscellaneous/WorkspaceIdleTimeout.spec.ts

```diff
@@ -46,7 +47,7 @@ suite('"Check workspace idle timeout" test', function (): void {
 	let stopWorkspaceTimeout: number = 0;
 
 	suiteSetup(function (): void {
-		kubernetesCommandLineToolsExecutor.loginToOcp('admin');
+		kubernetesCommandLineToolsExecutor.loginToOcp(OAUTH_CONSTANTS.TS_SELENIUM_OCP_USERNAME, OAUTH_CONSTANTS.TS_SELENIUM_OCP_PASSWORD);
```

Comment:
you could probably call `loginToOcp()`, because the method uses  OAUTH_CONSTANTS.TS_SELENIUM_OCP_USERNAME, OAUTH_CONSTANTS.TS_SELENIUM_OCP_PASSWORD values by default.

Reply:
Thanks

Reply:
Done

Reply:
Thanks